### PR TITLE
Release Notes for 128T Versions with Installer 3.0. This does not yet contain the installer-3.0 doc updates. 

### PR DESCRIPTION
**Pull request by Chr1st0ph3rTurn3r**
Installer 3.0 updates will be merged after that branch is released.

### CODE REVIEW

**Review by MichaelBaj — APPROVED** [review]

File: docs/release_notes_128t_4.5.md [hunk 1]
```diff
@@ -3,6 +3,21 @@ title: 128T 4.5 Release Notes
...
+
+### New Features and Improvements
+
+- **Support for Installer 3.0, providing token-based software access.** For Installation and Upgrade information, refer to [Conductor Interactive Installation](intro_installation_bootable_media.md). 
+
+### Resolved Issues
+
+- **I95-39650 Repository access tokens provisioned on the Conductor are not automatically distributed to its managed routers.** Access Tokens are now distributed to managed routers. 
+------
+- **I95-39649/BEL-42 Routers not going to running state when no release.pem (certificate).** Routers using token-based software access for installation or upgrade no longer look for both the certificate and the token.
```
Comment (jeffmart) [inline docs/release_notes_128t_4.5.md:16]: Lets reword this, the token suport is new so does not really apply from an end user perspective. Lets consider something like:

**I95-39649/BEL-42 Conductors/Routers on initial deployment not going to running state ** Conductors/Routers on initial deployment would not transition to a running state until a certificate was added.

File: docs/release_notes_128t_5.0.md [hunk 2]
```diff
@@ -3,6 +3,18 @@ title: 128T 5.0 Release Notes
...
+
+### New Features and Improvements
+
+- **Support for Installer 3.0, providing token-based software access.** For Installation and Upgrade information, refer to [Conductor Interactive Installation](intro_installation_bootable_media.md). 
+
+### Resolved Issues
+
+- **I95-39650 Repository access tokens provisioned on the Conductor are not automatically distributed to its managed routers.** Access Tokens are now distributed to managed routers. 
+------
+- **I95-39649/BEL-42 Routers not going to running state when no release.pem (certificate).** Routers using token-based software access for installation or upgrade no longer look for both the certificate and the token.
```
Comment (jeffmart) [inline docs/release_notes_128t_5.0.md:16]: Same as above

File: docs/release_notes_128t_5.1.md [hunk 3]
```diff
@@ -3,6 +3,18 @@ title: 128T 5.1 Release Notes
...
+
+### New Features and Improvements
+
+- **Support for Installer 3.0, providing token-based software access.** For Installation and Upgrade information, refer to [Conductor Interactive Installation](intro_installation_bootable_media.md). 
+
+### Resolved Issues
+
+- **I95-39650 Repository access tokens provisioned on the Conductor are not automatically distributed to its managed routers.** Access Tokens are now distributed to managed routers. 
+------
+- **I95-39649/BEL-42 Routers not going to running state when no release.pem (certificate).** Routers using token-based software access for installation or upgrade no longer look for both the certificate and the token.
```
Comment (jeffmart) [inline docs/release_notes_128t_5.1.md:16]: Same as above

**Review by jeffmart — CHANGES_REQUESTED** [review]

File: docs/release_notes_128t_4.5.md (hunk 1 above)
Comment (Chr1st0ph3rTurn3r) [reply docs/release_notes_128t_4.5.md]: done

File: docs/release_notes_128t_5.0.md (hunk 2 above)
Comment (Chr1st0ph3rTurn3r) [reply docs/release_notes_128t_5.0.md]: done

File: docs/release_notes_128t_5.1.md (hunk 3 above)
Comment (Chr1st0ph3rTurn3r) [reply docs/release_notes_128t_5.1.md]: done

**Review by jeffmart — APPROVED** [review]

**Review by MichaelBaj — APPROVED** [review]

**Pull request merged**